LABS.md

```diff
@@ -0,0 +1,57 @@
+# LoopBack Labs
+
+`Labs` is an incubation process to build experimental features and make it easy
+for our developers to try out via `npm install`. Such features are usually not
+production ready. The design, implementation and APIs may change significantly
+over versions. We may graduate or abandon experimental features based on the
+community feedback.
+
+## Workflow
+
+We use a convention based git branching strategy to work on experimental
+features along with regular ones in `loopback-next` repository.
+
+![loopback-labs](./labs/labs.png)
+
+### Work on an experimental feature
+
+1. Set up local git repository for an experiment feature
+
+```sh
+cd loopback-next
+git fetch --all
+git checkout labs
+git checkout -b labs/<my-experimental-feature>
```

Comment:
BTW, if branch `labs` exists, no branch named `labs/anything` can be created.

Reply:
I realized that. It's changed to `lab/*`.